R/ceteris_paribus.R

```diff
@@ -41,23 +41,21 @@
 #'
 #' explain_titanic_glm <- explain(model_titanic_glm,
 #'                                data = titanic_small[,-8],
-#'                                y = titanic_small[,8],
-#'                                verbose = FALSE)
+#'                                y = titanic_small[,8])
 #'
 #' cp_rf <- ceteris_paribus(explain_titanic_glm, titanic_small[1,])
 #' cp_rf
 #'
 #' plot(cp_rf, variables = "age")
 #'
 #' \donttest{
-#' library("randomForest")
-#' model_titanic_rf <- randomForest(survived ~.,  data = titanic_imputed)
+#' library("ranger")
+#' model_titanic_rf <- ranger(survived ~., data = titanic_imputed, probability = TRUE)
 #'
 #'
 #' explain_titanic_rf <- explain(model_titanic_rf,
 #'                               data = titanic_imputed[,-8],
 #'                               y = titanic_imputed[,8],
-#'                               label = "Random Forest v7",
```

Comment:
why this label is removed? it can be changed to Ranger v7

Reply:
I hope that "ranger forest" is ok.